homeassistant/components/ozw/light.py

```diff
@@ -140,29 +140,40 @@ def color_temp(self):
     def async_set_duration(self, **kwargs):
         """Set the transition time for the brightness value.
 
-        Zwave Dimming Duration values:
-        0       = instant
-        0-127   = 1 second to 127 seconds
-        128-254 = 1 minute to 127 minutes
-        255     = factory default
+        Zwave Dimming Duration values now use seconds as an
+        integer (max: 7620 seconds or 127 mins)
```

Comment:
Please add a link to the commit that is build 1205.

`https://github.com/OpenZWave/open-zwave/commit/f81bc04`

Reply:
Like so?
```
        """Set the transition time for the brightness value.

        Zwave Dimming Duration values now use seconds as an
        integer (max: 7620 seconds or 127 mins)
        Build 1205 https://github.com/OpenZWave/open-zwave/commit/f81bc04
        """
```

Reply:
Perfect!

Reply:
updated in next commit